### ISSUE TITLE
Weird errors when an instrumented function throws

### ISSUE THREAD

**Issue by Saikyun** [arcadia-unity/Arcadia#296]
When giving wrong arguments to a fdef'ed & instrumented function, you get a weird error.

```clojure
(require '[clojure.spec.alpha :as s]
         '[clojure.spec.test.alpha :as stest])
(s/fdef add
        :args (s/cat :n number?))
(defn add [n] (+ n n))
(stest/instrument `add)
(add 5)
;;user=> 10
(add "wat")
;; ArgumentException Don't know how to create ISeq from: System.Diagnostics.StackTrace
;; clojure.lang.RT.seqFrom (:0)
;; clojure.lang.RT.seq (:0)
;; clojure.lang.RT.iter (:0)
;; clojure/core$sequence__2566.invokeStatic (:0)
;; clojure/core$sequence__2566.invoke (:0)
;; clojure/spec/test/alpha$stacktrace_relevant_to_instrument__13706.invokeStatic (:0)
;; clojure/spec/test/alpha$stacktrace_relevant_to_instrument__13706.invoke (:0)
;; clojure/spec/test/alpha$spec_checking_fnconform_BANG___13712__13724.invoke (:0)
;; clojure/spec/test/alpha$spec_checking_fnfn__13727__13731.doInvoke (:0)
;; clojure.lang.RestFn.invoke (:0)
;; user$eval__13974__13979.invokeStatic (:0)
;; user$eval__13974__13979.invoke (:0)
;; clojure.lang.Compiler.eval (:0)
;; clojure.lang.Compiler.eval (:0)
;; clojure/core$eval__3112.invokeStatic (:0)
;; clojure/core$eval__3112.invoke (:0)
;; arcadia/socket_repl$game_thread_evalfn__12953fn__12958__12970.invoke (:0)
```

**Comment by Saikyun** [comment]
I don't know where the libraries are located, but this is my fix (in the file `arcadia/Libraries/clojure/spec/test/alpha.clj`):
```clojure
(defn- stacktrace-relevant-to-instrument
  "Takes a coll of stack trace elements (as returned by
  StackTraceElement->vec) and returns a coll of maps as per
  interpret-stack-trace-element that are relevant to a
  failure in instrument."
  [elems]
  (let [plumbing? (fn [{:keys [var-scope]}]
                    (contains? '#{clojure.spec.alpha.test/spec-checking-fn} var-scope))]
    (->> (.. elems GetFrames)
         (map StackTraceElement->vec)
         (map interpret-stack-trace-element)
         (filter :var-scope)
         (drop-while plumbing?))))
```

**Comment by Saikyun** [comment]
Btw I realize that my fix probably doesn't work exactly as intended, but as a hotfix to allow me to see the errors I want to see. :)

**Comment by timsgardner** [comment]
Which commit of Arcadia is this?

**Comment by Saikyun** [comment]
It's been like this since new 2018 version of Arcadia. I don't know exactly
which commit I'm on, I'll get back to you when I have time at the computer.

Den fre 31 aug. 2018 22:28Tims Gardner <notifications@github.com> skrev:

**Comment by Saikyun** [comment]
commit 182116afd25d19fd71065f2e2a6f94c7de3204da (HEAD -> develop, origin/develop, origin/HEAD)
Author: timsgardner <timsgardner@gmail.com>
Date:   Mon Aug 27 19:06:19 2018 -0400

    supplying socket-repl with injections.
    
    fixes #287

**Comment by timsgardner** [comment]
I think this got regressed when we changed how we're packaging up spec.

**Comment by timsgardner** [comment]
fixed as of b713ffebf781743752f369878a432b190630f552
